draft-ietf-quic-transport.md

```diff
@@ -73,9 +73,11 @@ informative:
     title: "Structured Streams: A New Transport Abstraction"
     author:
       - ins: B. Ford
-    date: 2007-08
+    date: 2007-10
```

Comment:
xml2rfc should be able to simply cite this paper by its DOI; see https://www.rfc-editor.org/pipermail/rfc-interest/2016-December/009805.html. (But I have no idea how to do this from kramdown.)